daemon/cmd/daemon_main.go

```diff
@@ -440,6 +440,9 @@ func initializeFlags() {
 	flags.Int64(option.ProxyIdleTimeout, 60, "Set Envoy upstream HTTP idle connection timeout seconds. Does not apply to connections with pending requests. Default 60s")
 	option.BindEnv(Vp, option.ProxyIdleTimeout)
 
+	flags.Uint(option.RestoredProxyPortsAgeLimit, 15, "Time after which a restored proxy ports file is considered stale (in minutes)")
+	option.BindEnv(vp, option.RestoredProxyPortsAgeLimit)
```

Comment:
Some compile failures, think we need to manually change this `vp` -> `Vp`

Reply:
Was not the first time I've been bitten by this... fixed!